src/app/styles/typography.scss

```diff
@@ -1,73 +1,81 @@
 @import './variables';
 
-$header-font-family: 'museo-sans', sans-serif;
-$default-font-family: 'Open Sans', sans-serif;
+@font-face {
+  font-family: 'museo';
+  src: url('./assets/fonts/museo/museo-300.otf') format("opentype");
+  font-weight: 300;
+}
 
-body {
-  h1 {
-    font-family: $header-font-family;
-    font-size: 42px;
-    font-weight: 700;
-  }
+@font-face {
+    font-family: 'museo';
+    src: url('./assets/fonts/museo/museo-500.otf') format("opentype");
+    font-weight: 500;
+}
 
-  h2 {
-    font-family: $header-font-family;
-    font-size: 36px;
+@font-face {
+    font-family: 'museo';
+    src: url('./assets/fonts/museo/museo-700.otf') format("opentype");
     font-weight: 700;
-  }
+}
 
-  h3 {
-    font-family: $header-font-family;
-    font-size: 24px;
-    font-weight: 700;
-  }
+// font stacks
+$museo:                "museo", "Helvetica Neue", Helvetica, Arial, sans-serif;
+$default-font-family:  "Open Sans", sans-serif;
 
-  h4 {
-    font-family: $header-font-family;
-    font-size: 18px;
-    font-weight: 700;
-  }
 
-  h5 {
-    font-family: $header-font-family;
-    font-size: 16px;
-    font-weight: 700;
-  }
+h1, h2, h3, h4, h5, h6 {
+  font-family: $museo !important;
```

Comment:
Why the need for `!important`? 

 Personally  don't think we should need it on such a general selector or am I missing something? 

_Sidenote_: We had a rule where I worked before that I really liked which was that every usage of `!important` needed to be commented/documented.